source/com.google.android.gms/play-services-mlkit-barcode-scanning/Transforms/Metadata.Namespaces.xml

```diff
@@ -7,4 +7,27 @@
         Package
     </attr>
     -->
+    <attr 
+        path="/api/package[@name='com.google.android.gms.internal.mlkit_vision_barcode']" 
+        name="managedName"
+        >
+        Android.Gms.Internal.MLKitVisionBarCode
+    </attr>
+    <attr 
+        path="/api/package[@name='com.google.mlkit.vision.barcode']" 
+        name="managedName"
+        >
+        Android.Gms.MLKit.Vision.BarCode
```

Comment:
In https://github.com/xamarin/GooglePlayServicesComponents/pull/421, the namepace for `com.google.mlkit...` was changed to `Xamarin.Google.MLKit...`.  Here it is `Android.Gms.MLKit...`.

Is there a reason for the inconsistency?

Reply:
these packages from 2 different groupIds.

I know- confusing as hell!

Reply:
Seem like they plan to move it from GMS to MLKit. 
I think you are right.
I hope people will not use both of those packages... Type collisions are expected to happen